client/client.go

```diff
@@ -45,7 +47,7 @@ type ClientInfo struct {
 	Meta             map[string]interface{}
 }
 
-func (cli *Client) call(ctx context.Context, paramsRawVals []interface{}, interfaceName, methodName, callType string, opts ...CallOption) (protocol.Result, error) {
+func (cli *Client) call(ctx context.Context, paramsRawVals []interface{}, interfaceName, methodName, group, version, callType string, opts ...CallOption) (protocol.Result, error) {
```

Comment:
We need to consider that whether ```group``` and ```version``` should be required fields. 
Since this function is relevant to non-idl API, we need to keep it simple.
For instance, if users do not need to configure ```group``` and ```version```,
```
cli.CallUnary(ctx, req, resp, interfaceName, methodName, "", "") 
```
Empty strings are very redundant.
I prefer to use ```CallOption``` to carry ```group``` and ```version```.

Reply:
Good point. I was also thinking of using CallOption in the first place. I will move group and version to CallOption later.

Reply:
Putting group and version into CallOption has other issues. But still it's currently the best way I can think of.